pkgs/development/python-modules/gst-python/default.nix

```diff
@@ -28,22 +28,29 @@ buildPythonPackage rec {
   # Python 2.x is not supported.
   disabled = !isPy3k;
 
+  depsBuildBuild = [
+    pkg-config
+  ];
+
   nativeBuildInputs = [
     meson
     ninja
     pkg-config
-    python
     gobject-introspection
-    gst-plugins-base
+    gst_all_1.gst-plugins-base
   ];
 
   propagatedBuildInputs = [
-    gst-plugins-base
+    gst_all_1.gst-plugins-base
     pygobject3
   ];
 
   mesonFlags = [
     "-Dpygi-overrides-dir=${placeholder "out"}/${python.sitePackages}/gi/overrides"
+    # This is only used for figuring out what version of Python is in
+    # use, and related stuff like figuring out what the install prefix
+    # should be, but it does need to be able to execute Python code.
+    "-Dpython=${python.pythonForBuild.interpreter}"
```

Comment:
This is wrong, the python interpreter is being linked against: c3e6d630dc29101a25a383cb530e7aa3c002/subprojects/gst-python/plugin/meson.build#L4

Reply:
changed the comment